lib/dialects/abstract/query-generator.js

```diff
@@ -1452,7 +1452,7 @@ class QueryGenerator {
           ? this.quoteAttribute(attr, options.model)
           : this.escape(attr);
       }
-      if (!_.isEmpty(options.include) && !attr.includes('.') && addTable) {
+      if (!_.isEmpty(options.include) && (!attr.includes('.') || options.dotnotation) && addTable) {
```

Comment:
Can you please rename it to `dotNotation`?

Reply:
Sure